app/scripts/modules/core/delivery/service/executions.transformer.service.js

```diff
@@ -205,9 +205,6 @@ module.exports = angular.module('spinnaker.core.delivery.executionTransformer.se
     }
 
     function applyPhasesAndLink(execution) {
-      if (!execution.parallel) {
-        return;
-      }
```

Comment:
Don't want to remove this just yet, unless you're converting all the existing execution history to the parallel structure, too (in which case this is fine to remove)